cms/envs/common.py

```diff
@@ -2236,3 +2236,6 @@
 
 ##### LOGISTRATION RATE LIMIT SETTINGS #####
 LOGISTRATION_RATELIMIT_RATE = '500/5m'
+
+######################## Setting for content libraries ########################
+MAX_BLOCK_COUNT = 1000
```

Comment:
I think this should have a more specific name like `MAX_BLOCKS_PER_CONTENT_LIBRARY`

`MAX_BLOCK_COUNT` could be referring to courses, some kind of cache, the system as a whole, or anything.

Reply:
Thanks for noticing that - fixed!